cli/args/flags.rs

```diff
@@ -1032,13 +1031,12 @@ fn clap_root() -> Command {
         .global(true),
     )
     .arg(
+      // NOTE(bartlomieju): not used anymore, but kept for CLI compatibility between
```

Comment:
I think it's not too big of a deal to just remove this? It's unstable and not a lot of people are using it.

Reply:
Sure I can do that.